asserter/asserter.go

```diff
@@ -46,10 +46,11 @@ type Asserter struct {
 // on the transaction. Fore more details please refer to
 // https://github.com/coinbase/rosetta-sdk-go/tree/master/asserter#readme
 type Validations struct {
-	Enabled   bool                 `json:"enabled"`
-	ChainType ChainType            `json:"chain_type"`
-	Payment   *ValidationOperation `json:"payment"`
-	Fee       *ValidationOperation `json:"fee"`
+	Enabled    bool                 `json:"enabled"`
+	RelatedOps bool                 `json:"related_ops"`
```

Comment:
Should we rename it something like `related_op_present` or `related_op_exists`? or something which sounds more like a boolean? just `RelatedOps` is not clear of what that param is.